src/configuration/services/headless-chrome.md

```diff
@@ -0,0 +1,158 @@
+# Headless Chrome
+
+Headless Chrome is a headless browser that can be configured on projects like any other service on Platform.sh. You can interact with the `headless-chrome` service container using Puppeteer, a Node library that provides an API to control Chrome over the DevTools Protocol. 
+
+Puppeteer can be used to generate PDFs and screenshots of web pages, automate form submission, and test your project's UI. You can find out more information about using Puppeteer on [GitHub](https://github.com/GoogleChrome/puppeteer) or in their [documentation](https://pptr.dev/).
+
+## Supported versions
+
+* 73
+
+## Relationship
+
+The format exposed in the `$PLATFORM_RELATIONSHIPS` [environment variable](/development/variables.md#platformsh-provided-variables):
+
+```yaml
+{
+    "service": "headless",
+    "ip": "169.254.73.96",
+    "hostname": "3rxha4e2w4yv36lqlypy7qlkza.headless.service._.eu-3.platformsh.site",
+    "cluster": "moqwtrvgc63mo-master-7rqtwti",
+    "host": "headless.internal",
+    "rel": "http",
+    "scheme": "http",
+    "type": "chrome-headless:73",
+    "port": 9222
+}
+
+```
+
+## Requirements
+
+Puppeteer requires at least Node.js version 6.4.0, while using the async and await examples below requires Node 7.6.0 or greater. Using the Platform.sh Config Reader library requires Node.js 10 or later.
+
+## Usage example
+
+In your `.platform/services.yaml`:
+
+```yaml
+headless:
+  type: chrome-headless:73
+```
+
+In your `.platform.app.yaml`:
+
+```yaml
+relationships:
+  headless: "headless:http"
+```
+
+After configuration, include Puppeteer as a dependency in your `package.json`:
+
+```json
+"puppeteer": "^1.14.0"
+```
+
+Using the [Node.js Config Reader](https://github.com/platformsh/config-reader-nodejs) library and Puppeteer, you can define credentials for connecting to headless Chrome using the `browserURL` parameter of `puppeteer.connect()`.
+
+### Taking screenshots
```

Comment:
I think these examples should be How-to guides on Community as they are not reference material.